src/sagas/transactions/boarsagas.js

```diff
@@ -1,17 +1,87 @@
 import { AsyncStorage } from 'react-native';
-import { TRANSACTION_FOUND } from './constants';
 import { SYMBOL_HOARD } from 'containers/App/constants';
-import { call, put } from 'redux-saga/effects';
+import { call, put, takeEvery } from 'redux-saga/effects';
 import { bigNumberToEther } from 'lib/formatters';
 import { TYPE_SEND, TYPE_REQUEST } from 'screens/SendRequest/constants';
+import {
+  transactionFound,
+  searchForTransactions,
+} from 'sagas/transactions/actions';
+import {
+  WALLET_IMPORT_SUCCESS,
+  WALLET_TRACK_SYMBOL_SUCCESS,
+} from 'screens/Wallet/constants';
 
-import { provider, getBlock } from 'sagas/transactions/ethsagas';
+import { updateBalance } from 'screens/Wallet/actions';
+import { provider } from 'sagas/transactions/ethsagas';
+import { actionBridgeChannel } from 'sagas/transactions/helpers';
+
+import { getWalletInstance } from 'screens/Wallet/sagas';
 
 export const LAST_HOARD_BLOCK_STORAGE_KEY = 'LAST_HOARD_BLOCK_STORAGE_KEY';
 
 const HOARD_BASE_PRICE = 0.035;
 
-export function* fetchHistoryBoar({ wallet }) {
+export default function* boarTransactionsSagaWatcher() {
```

Comment:
Very very minor `boar -> hoard`
```suggestion
export default function* hoardTransactionsSagaWatcher()
```